package.json

```diff
@@ -54,7 +54,7 @@
     "react-chartjs-2": "2.8.0",
     "react-copy-to-clipboard": "5.0.1",
     "react-datetime": "2.16.3",
-    "react-dom": "16.8.4",
+    "react-dom": "16.10.2",

```

Comment:
What's the reason to update?
Old version crashes while running tests?

Reply:
Yep, at least here, this had add new warnings in the browser console, warnings to update the way a few things are done

Reply:
@iguit0 the erros where fixed, please review the merge again.